l10n_it_ricevute_bancarie/models/account/account.py

```diff
@@ -194,19 +196,21 @@ def action_move_create(self):
             # ---- Apply Collection Fees on invoice only on first due date of the month
             # ---- Get Date of first due date
             move_line = self.env['account.move.line'].search([
-                ('partner_id', '=', invoice.partner_id.id)])
+                ('partner_id', '=', invoice.partner_id.id),
+                ('invoice_id.payment_term_id.riba', '=', True)
```

Comment:
Può essere il caso di controllare che `invoice_id.payment_term_id` sia `!= False` anche? O è required e quindi sappiamo esserci sempre?

Reply:
Non sono certo di questa cosa. Andrebbe fatto un test.

Reply:
Se non viene specificato il termine di pagamento, non è possibile emettere riba

Reply:
@OpenCode aggiungo che poco prima questa ricerca (riga 188) c'è un controllo che evita di prendere in considerazione fatture che non hanno il termine di pagamento o che se ce l'hanno non siano di tipo riba.
Quindi penso che in questa domain non debbano essere fatti ulteriori controlli